packages/sandbox/src/components/AssetCardList/AssetCardListConfig.ts

```diff
@@ -66,11 +62,15 @@ export const AssetCardListConfig: Array<CardProps> = [
     name: 'Avalanche',
     icon: 'avax.png',
     symbol: 'AVAX',
-    enabled: false,
+    enabled: true,
```

Comment:
sandbox so doesn't really matter for the runtime snap / adapter but do we want these enabled?

Reply:
Also this is commented so should probably be false either way

Reply:
One of the security review requests is to provide sandbox support for all chains supported by the snap. Not sure yet if there's consensus agreement on a canonical list.

Reply:
The snap doesn't support AVAX though? MetaMask does

Reply:
Also this is commented regardless, so I'm not sure how this tackles sandbox support